plugins/modules/identity/ipa/ipa_dnszone.py

```diff
@@ -32,6 +32,12 @@
     default: "false"
     choices: ["false", "true"]
     type: str
+  allowsyncptr:
+    description: Allow synchronization of forward and reverse records in the zone
+    required: false
+    default: "false"
```

Comment:
New state options should never have a default value, to avoid breaking backwards compatibility. If not specified, they should be ignored for idempotency checks.

Reply:
Agreed - while the code does not actually support a "default" value, I wanted to keep docs in following the "standard" that's been set? I agree it is confusing, so I removed this.